mmdet/datasets/dataset_wrappers.py

```diff
@@ -167,3 +167,13 @@ def update_skip_type_keys(self, skip_type_keys):
             isinstance(skip_type_key, str) for skip_type_key in skip_type_keys
         ])
         self._skip_type_keys = skip_type_keys
+
+    def has_all_skip_type_keys(self, skip_type_keys):
```

Comment:
This will result in backwards compatibility breaking.

Reply:
" think there is a simpler solution, which is to introduce a 'has_switch' flag in this hook to handle it.

Reply:
@hhaAndroid  Thank you for your review. I fixed this.